app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -78,33 +68,30 @@ protected void onCreate(Bundle savedInstanceState) {
             e.printStackTrace();
         }
         bnv.setOnNavigationItemSelectedListener(
-                new BottomNavigationView.OnNavigationItemSelectedListener() {
-                    @Override
-                    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-                        FragmentTransaction transaction =
-                                getSupportFragmentManager().beginTransaction();
-                        switch (item.getItemId()) {
-                            case R.id.action_home:
-                                mFirebaseAnalytics.logEvent("eatery_tab_press", null);
-                                transaction
-                                        .replace(R.id.frame_fragment_holder, mainListFragment)
-                                        .commit();
-                                break;
-                            case R.id.action_week:
-                                mFirebaseAnalytics.logEvent("lookahead_tab_press", null);
-                                transaction
-                                        .replace(R.id.frame_fragment_holder, weeklyMenuFragment)
-                                        .commit();
-                                break;
-                            case R.id.action_brb:
-                                mFirebaseAnalytics.logEvent("brb_tab_press", null);
-                                transaction
-                                        .replace(R.id.frame_fragment_holder, loginFragment)
-                                        .commit();
-                                break;
-                        }
-                        return true;
+                (@NonNull MenuItem item) -> {
+                    FragmentTransaction transaction =
+                            getSupportFragmentManager().beginTransaction();
+                    switch (item.getItemId()) {
+                        case R.id.action_home:
+                            mFirebaseAnalytics.logEvent("eatery_tab_press", null);
+                            transaction
+                                    .replace(R.id.frame_fragment_holder, mainListFragment)
+                                    .commit();
+                            break;
+                        case R.id.action_week:
+                            mFirebaseAnalytics.logEvent("lookahead_tab_press", null);
+                            transaction
+                                    .replace(R.id.frame_fragment_holder, weeklyMenuFragment)
+                                    .commit();
+                            break;
+                        case R.id.action_brb:
+                            mFirebaseAnalytics.logEvent("brb_tab_press", null);
```

Comment:
Move log strings to assets

Reply:
ye same reasoning here Android Studio didn't complain about this because the strings are internal and never displayed to the user so they won't need to be translated